smartsim/ml/torch/data.py

```diff
@@ -28,10 +28,10 @@
 
 import numpy as np
 import torch
-from smartredis import Client, Dataset
```

Comment:
Ditto "shouldn't matter, but symbols are no longer defined and being used" comment here. Consider setting up some mocks or removing the usage to get past the CI.